packages/sdk/src/chains/EVM/assetTransfer.ts

```diff
@@ -191,17 +193,39 @@ export class EVMAssetTransfer extends BaseAssetTransfer {
    * Builds an unsigned transfer transaction.
    * Should be executed after the approval transactions.
    *
-   * @param transfer
-   * @param fee
+   * @param {Transfer} transfer requested transfer
+   * @param {Fee} fee Fee calculated by 'getFee' function
+   * @param {Boolean} skipDestinationBalanceCheck Flag to disable destination chain balance check
+   * @param {String} destinationProviderUrl URL for destination chain provider
    * @returns unsigned transfer transaction
+   * @throws {Error} Destination Chain URL is required
+   * @throws {Error} Insufficient destination chain liquidity to proceed with transfer
    */
   public async buildTransferTransaction(
     transfer: Transfer<TransferType>,
     fee: EvmFee,
+    skipDestinationBalanceCheck: boolean = false,
```

Comment:
This is extra.
If the destinationProviderUrl is undefined it means the same thing